client/cli/go/glide.yaml

```diff
@@ -0,0 +1,15 @@
+package: github.com/heketi/heketi/client/cli/go
+import:
+- package: github.com/heketi/heketi
```

Comment:
Is this package necessary?  Pulling the source?  I could see this causing issues if someone adds a file.